experiments/mjc_pointmass_observed/hyperparams.py

```diff
@@ -0,0 +1,163 @@
+""" Hyperparameters for MJC 2D navigation policy optimization. """
```

Comment:
I think this is the obstacle experiment in the paper and so are the following two files.